docs/api.md

```diff
@@ -33,6 +33,8 @@ mainly, but not only, provisioning details.
   * *credentialsName* -- A reference to a *secret* containing the
     username and password for the BMC.
 
+  * *disableCertificateVerification* -- A boolean to skip certificate validation when true.
```

Comment:
I'm believe this line needs to be wrapped to 80 characters to pass the markdown linting test.

Reply:
Thanks I pushed a fix for this